src/mmw/js/src/analyze/views.js

```diff
@@ -57,6 +57,33 @@ var ResultsView = Marionette.LayoutView.extend({
             newProjectUrl = '/project/new/' + modelPackageName,
             projectUrl = '/project';
 
+        if (modelPackageName === 'gwlfe') {
+            var analysisResults = JSON.parse(App.getAnalyzeCollection()
+                                                .findWhere({taskName: 'analyze'})
+                                                .get('result'));
+
+            var landResults = _.find(analysisResults, function(element) {
```

Comment:
In MMW there is a convention of grouping variable declarations together in JavaScript. Thus, prefer:

```js
var analysisResults = JSON.parse(App.getAnalyzeCollection()
                                    .findWhere({taskName: 'analyze'})
                                    .get('result')),
    landResults = _.find(analysisResults, function(element) {
        return element.name === 'land';
    });
```